packages/account/src/Components/poinc/status/limited/limited.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Icon } from '@deriv/components';
+import { localize, Localize } from '@deriv/translations';
+import IconMessageContent from 'Components/icon-message-content';
+
+export const PoincLimited = () => (
+    <IconMessageContent
```

Comment:
This is exactly the same component as `poi/status/limited`, only icon is different. Please reuse the same component instead, make it universal and delete this.